config/install/autosave_form.settings.yml

```diff
@@ -0,0 +1,12 @@
+interval: 10000
```

Comment:
This configuration file might not get imported and throw an error. Please check.